addons/mail/static/src/messaging/test_utils.js

```diff
@@ -681,9 +681,10 @@ async function start(param0 = {}) {
         await testEnv.messagingInitializedPromise;
     }
 
-    await Promise.all(mountCallbacks.map(callback => callback({ selector, widget })));
-    if (hasChatWindow || hasDiscuss || hasMessagingMenu) {
-        await afterNextRender();
+    if (mountCallbacks.length > 0) {
+        await afterNextRender(() => {
+          Promise.all(mountCallbacks.map(callback => callback({ selector, widget })));
```

Comment:
indent, keep await

Reply:
ok for indent but what do you mean by "keep await" ?

Reply:
`await` before `Promise.all()` (turn arrow function `async`)

Reply:
There was an `await` before `Promise.all` in the original code, it needs to be kept. `afterNextRender` is doing `await` on the passed function to ensure the full processing is done before notifying the end of the render (basically if there are multiple following renders it ensures to wait the end of the last one).